ui/packages/catalog-ui-search/src/main/webapp/component/tabs/metacards/tabs-metacards.js

```diff
@@ -10,11 +10,23 @@
  *
  **/
 /*global define*/
-const _ = require('underscore')
+import React from 'react'
 const Tabs = require('../tabs')
-const store = require('../../../js/store.js')
+const Marionette = require('marionette')
 const MetacardsBasicView = require('../../editor/metacards-basic/metacards-basic.view.js')
-const MetacardArchiveView = require('../../metacard-archive/metacard-archive.view.js')
+import MetacardArchive from '../../../react-component/container/metacard-archive'
+
+const MetacardArchiveView = Marionette.LayoutView.extend({
```

Comment:
What is the difference between the following files? They look identical.

`tabs/metacards/tabs-metacards.js`
`tabs/metacard/tabs-metacard.js`

Reply:
`metacards.js` is used for multiselect which only supports archive and details tabs